src/helm/proxy/clients/openai_client.py

```diff
@@ -19,6 +21,7 @@
 
 class OpenAIClient(Client):
     END_OF_TEXT: str = "<|endoftext|>"
+    MODELS_USING_TIKTOKEN: List[str] = ["gpt-3.5-turbo-0301", "gpt-4-32k-0314"]
```

Comment:
We probably don't need `MODELS_USING_TIKTOKEN` if the other changes are implemented.

Reply:
So I am not sure about reproducibility now. I feel like we are calling everytime `openai/ + tokernizer_name` in completion while for some models we should still be using `huggingface/gpt2`.